tables_app/src/main/java/org/opendatakit/tables/utils/Constants.java

```diff
@@ -337,4 +337,18 @@ private JavaScriptHandles() {
     }
   }
 
+    public enum TABLE_SORT_ORDER
+    {
+      SORT_ASC, SORT_DESC;
+
+      public String getCorrespondingName() { return this.name(); }
+      public static TABLE_SORT_ORDER getCorrespondingEnum(String table_sort_order)
+      {
+        if( table_sort_order.equalsIgnoreCase( SORT_ASC.name() ) )
+          return SORT_ASC;
+        else
+          return SORT_DESC;
+      }
```

Comment:
`getCorrespondingName` and `getCorrespondingEnum` are already provided by Java, Enum.toString() and Enum.valueOf(), respectively.